src/rules/rule-non-nested-empty-line-before/README.md

```diff
@@ -132,3 +132,41 @@ a
 b
 {}
 ```
+
+## Optional options
+
+### `except: ["after-single-line-comment"]`
+
+Reverse the primary option if the rule is after a single line comment.
+
+For example, with `"always"`:
+
+The following patterns are considered warnings:
+
+```css
+/**
```

Comment:
Yes, updated in https://github.com/stylelint/stylelint/pull/1543/commits/6eb38a62fdaaf6c3fced60180fc3ffc31354e84e